examples/DocExampleApis/Program.cs

```diff
@@ -122,4 +137,134 @@ public static async Task Example_API_Delete(CacheClient cacheClient)
+        }
+        else if (keyPrefixAllCachesToken is GenerateDisposableTokenResponse.Error err)
+        {
+            Console.WriteLine("Error generating disposable token: " + err.Message);
+        }
+
+        // Generate a disposable token with read-only access to all topics in one cache
+        var allTopicsOneCacheToken = await authClient.GenerateDisposableTokenAsync(
+            DisposableTokenScopes.TopicSubscribeOnly(CacheSelector.ByName("squirrel"), TopicSelector.AllTopics),
+            ExpiresIn.Minutes(30)
+        );
+        if (allTopicsOneCacheToken is GenerateDisposableTokenResponse.Success token3)
+        {
+            // logging only a substring of the tokens, because logging security credentials is not advisable :)
+            Console.WriteLine("The generated disposable token starts with: " + token3.AuthToken.Substring(0, 10));
+            Console.WriteLine("The token expires at (epoch timestamp): " + token3.ExpiresAt.Epoch());
+        }
+        else if (allTopicsOneCacheToken is GenerateDisposableTokenResponse.Error err)
+        {
+            Console.WriteLine("Error generating disposable token: " + err.Message);
+        }
+
+        // Generate a disposable token with write-only access to a single topic in all caches
+        var oneTopicAllCachesToken = await authClient.GenerateDisposableTokenAsync(
+            DisposableTokenScopes.TopicPublishOnly(CacheSelector.AllCaches, "acorn"),
+            ExpiresIn.Minutes(30)
+        );
+        if (oneTopicAllCachesToken is GenerateDisposableTokenResponse.Success token4)
+        {
+            // logging only a substring of the tokens, because logging security credentials is not advisable :)
+            Console.WriteLine("The generated disposable token starts with: " + token4.AuthToken.Substring(0, 10));
+            Console.WriteLine("The token expires at (epoch timestamp): " + token4.ExpiresAt.Epoch());
+        }
+        else if (oneTopicAllCachesToken is GenerateDisposableTokenResponse.Error err)
+        {
+            Console.WriteLine("Error generating disposable token: " + err.Message);
+        }
+
+    }
+
+    public static async Task Example_API_InstantiateTopicClient()
+    {
+        new TopicClient(
+            TopicConfigurations.Laptop.latest(), 
+            new EnvMomentoTokenProvider("MOMENTO_AUTH_TOKEN")
+        );
+    }
+    public static async Task Example_API_TopicPublish(ITopicClient topicClient)
+    {
+       var publishResponse =
+            await topicClient.PublishAsync("test-cache", "test-topic", "test-topic-value");
+        switch (publishResponse)
+        {
+            case TopicPublishResponse.Success:
+                Console.WriteLine("Successfully published message to 'test-topic'");
+                break;
+            case TopicPublishResponse.Error error:
+                throw new Exception($"An error occurred while publishing topic message: {error.ErrorCode}: {error}");
+                break;

```

Comment:
I don't think we need a break here after the exception. Looks like there are a couple of these.